Editor/Editors/CinemachineTrackedDollyEditor.cs

```diff
@@ -54,11 +54,10 @@ private static void DrawTrackeDollyGizmos(CinemachineTrackedDolly target, GizmoT
         {
             if (target.IsValid)
             {
-                CinemachinePathBase path = target.m_Path;
+                CinemachinePathCache path = target.m_PathCache;
                 if (path != null)
                 {
                     var isActive = CinemachineCore.Instance.IsLive(target.VirtualCamera);
-                    CinemachinePathEditor.DrawPathGizmo(path, path.m_Appearance.pathColor, isActive);
```

Comment:
I remove the `Appearance` here since I think it should be a part of `SplineContainer`. I will talk to the spline team to see if I can contributes some codes there. If they feel it doesn't make sense to add code only used in `cinemachine` I will be back to create some extensions.